recipe/build.sh

```diff
@@ -137,7 +137,7 @@ if [[ "$host_platform" == "$target_platform" ]]; then
   rm ${PREFIX}/lib/libgfortran.spec
   sed "s#@CONDA_PREFIX@#$PREFIX#g" $RECIPE_DIR/libgfortran.spec > ${PREFIX}/lib/libgfortran.spec
 
-  for file in libgfortran.spec libgomp.spec libgfortran.a libgomp.a; do
```

Comment:
I guess gcc is smart enough to know that if it cannot find the `.a` file to link to the `.so`?

Reply:
yes